orbit/pkg/packaging/linux_shared.go

```diff
@@ -40,7 +40,12 @@ func buildNFPM(opt Options, pkger nfpm.Packager) (string, error) {
 	updateOpt := update.DefaultOptions
 
 	updateOpt.RootDirectory = orbitRoot
-	updateOpt.Targets = update.LinuxTargets
+	envGoArch := os.Getenv("GOARCH")
```

Comment:
I was expecting we'd use `--type`, so users would run something like `fleetctl package --type=deb-arm64...` or `fleetctl package --type=rpm-arm64 ...`. If this was already discussed with the product team then I'm good with this.

Reply:
It was not discussed. I can absolutely take that path for creating cross-platform packages 🙂

Reply:
I was just suggesting using `--type` instead asking the user to use an environment variable (`GOARCH`). We should check with the product team.

Reply:
good point, it was informally discussed, my understanding is that the sentiment is it should work similar to macOS (no extra config, a bit of magic)

let's bring it to stand-up today and document what we learn.

Reply:
yeah, I got you! It's just that the suggestion during estimation (IIRC) was to build some hack that resembles fat macOS binaries (eg: a bash script that downloads the right thing based on the host arch)

Reply:
yeah, that's why I mentioned they wanted to do some kind of hack (eg: a script) that's probably why this is not specified on the issue.

let's bring this up in stand-up, I believe George and Marko can share what they had in mind

Reply:
It might also be worth looking into an `--arch` flag instead

Reply:
So we would use `--type deb --arch arm64` instead of `--type deb-arm64`.